blockchain_connector/docker/Dockerfile-ethereum

```diff
@@ -162,12 +168,25 @@ COPY ./sdk/avalon_sdk/tcf_connector.toml /project/avalon/sdk/avalon_sdk/
 # Installing wheel file requires python3-pip package.
 # python3-pip package will increase size of final docker image.
 # So remove python3-pip package and dependencies.
-RUN apt-get update \
- && apt-get install -y -q python3-pip \
- && echo "Install Common Python, SDK & Blockchain_connector packages\n" \
- && pip3 install dist/*.whl \
- && pip3 install pycryptodomex ecdsa \
- && echo "Remove unused packages from image\n" \
- && apt-get autoremove --purge -y -q python3-pip \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/*
+RUN packages="python3-pip"; \
+    pip_packages="pycryptodomex ecdsa"; \
+    if [ "$DISTRO" = "centos" ] ; then \
+      packages="$packages which"; \
+    fi; \
+    /project/avalon/scripts/install_packages -c install -q "$packages" -p "$pip_packages" \
+    && echo "Install Common Python, SDK & Blockchain_connector packages\n" \
+    && pip3 install dist/*.whl \
+    && echo "Remove unused packages from image\n" \
+    && /project/avalon/scripts/install_packages -c uninstall -q "$packages" \
+    && if [ "$DISTRO" = "centos" ] ; then \
+         packages="openssl-devel wget make gcc"; \
+         /project/avalon/scripts/install_packages -c install -q "$packages"; \
+         cd /tmp \
+         && wget https://www.python.org/ftp/python/3.6.9/Python-3.6.9.tgz \
```

Comment:
Do we want to stick to minor version (3.6.**9**)? Would this be consistent elsewhere?

Reply:
in bionic, it found that it installs 3.6.9 and so it found _sha256 package. But in centos, by defaults it installs some older version of python. So, need to install this version to overcome _sha256 module unavailability.